app.js

```diff
@@ -4,22 +4,28 @@ dotenv.config();
 import bodyParser from 'body-parser';
 import { getUser, getFavorites, userLogin, userSignup, addFavorite, initialize } from './routes/user.js';
 import { getComments, addComment } from './routes/comments.js';
+import morgan from 'morgan';
 
 const app = express();
 const PORT = process.env.PORT || 8080;
 //Middleware to pass json body types in request
 const jsonParser = bodyParser.json();
+//Morgan token for use and custom tokens
+morgan.token('body', (req) => JSON.stringify(req.body));
+morgan.token('id', (req) => req.params.id);
+morgan.token('movieId', (req) => req.params.movieId);
+app.use(morgan(':url :method :status'));
```

Comment:
Du kan bare bruge tiny.
```suggestion
app.use(morgan('tiny'));
```